src/python/zquantum/qcbm/ansatz_utils.py

```diff
@@ -0,0 +1,117 @@
+from zquantum.core.circuit import Circuit, Gate, Qubit
+import numpy as np
+from typing import List
+
+
+def get_single_qubit_layer(
```

Comment:
We already have a method for that in `core.circuit.utils` AFAIR.

Reply:
Just seeing `create_layer_of_gates` now, but it looks as if it doesn't handle parameterized gates. We should probably go and add optional support there for parameterized gates. Not sure if this is something that should be done now or wait for later?

Reply:
Addressed here: https://github.com/zapatacomputing/z-quantum-core/pull/54/files and in this commit: f546e83